packages/server/src/lib/history.ts

```diff
@@ -52,17 +52,34 @@ export const getArticleViewCountByTagForWeeks = (
     }, 0);
 };
 
+export const getArticleViewCountByMultipleTagForWeeks = (
```

Comment:
can the existing `getArticleViewCountByTagForWeeks` function be removed?

Reply:
Removed getArticleViewCountByTagForWeeks